### DISCUSSION TITLE
Unable to run the code. Help Required!

### DISCUSSION THREAD

**Discussion by darkcheftar** [jzillmann/pdf-to-markdown#26] [Q&A] [answered]
I tried cloning the repository onto my local machine in order to look into issue #25. When I tried to run the code on my machine it's giving an error
![image](https://user-images.githubusercontent.com/54063851/113094702-cad42e80-920f-11eb-833e-5a37984a99ae.png).
Can anyone please help me in solving the issue

**Comment by jzillmann** [comment, marked as answer]
Hey @darkcheftar , nice that you wanna jump on #25!

First info, the area where this is needed is in a different branch. It's in https://github.com/jzillmann/pdf-to-markdown/tree/modularize a rewrite which is happening. Also despite the old code still being there, the development happens in the `core` and `ui` subfolders.

So into the `ui` folder, run `npm` start and http://localhost:3005/ will open.
Hope it works (this isn't tested on windows or linux).
Also note that I switched the UI framework from `React` to `Svelte`.

Let me know if you need more info!

**Reply by darkcheftar** [reply]
Finally **Up and running**, Thanks for the clarification. sire @jzillmann 
![image](https://user-images.githubusercontent.com/54063851/113135765-25857e80-9240-11eb-958b-5752c7abbe70.png)

**Comment by darkcheftar** [comment]
Hey @jzillmann I want to work on #17, 
As far as I understand I need to change work on master branch.
When I try to run the code I'm getting into the below error I mentioned.
Let me know, How Can I resolve this, or correct me If I'm missing something
![image](https://user-images.githubusercontent.com/54063851/221504655-37a8b19c-e18f-45d2-b19c-4521027b7022.png)
